CRM/Core/CodeGen/I18n.php

```diff
@@ -21,7 +21,11 @@ public function generateInstallLangs() {
     }
     // @todo Somewhere in 2024 or beyond, deprecate the old installer
     file_put_contents('../install/langs.php', "<?php \$langs = " . var_export($langs, TRUE) . ";");
-    file_put_contents('../setup/res/languages.php', "<?php\n\n\Civi\Setup::assertRunning();\n\n\$langs = " . var_export($langs, TRUE) . ";");
+
+    // The `Template` helper provides PHP code cleanup.
+    $tpl = new CRM_Core_CodeGen_Util_Template('php');
+    $tpl->assign('langs', var_export($langs, TRUE));
+    $tpl->run('languages_setup.tpl', '../setup/res/languages.php');
```

Comment:
@totten is there a file that maybe didn't get committed here? What is languages_setup.tpl?

I'm getting an error running regen.

Reply:
Doh! Thanks @demeritcowboy. Added #28209.